rageshake.sample.yaml

```diff
@@ -12,6 +12,16 @@ listings_auth_pass: secret
 # An empty or missing list will retain legacy behaviour and permit reports from any application name.
 allowed_app_names: []
 
+# If any submission matches one of these rejection conditions, the submission is rejected.
```

Comment:
It would be good to mention here that `app` is required, but `version` and `label` are both optional.